waspc/examples/todo-typescript/src/MainPage.tsx

```diff
@@ -0,0 +1,106 @@
+import './Main.css'
+import React, { useEffect, FormEventHandler, FormEvent } from 'react'
+import logout from '@wasp/auth/logout'
+import useAuth from '@wasp/auth/useAuth'
+import { useQuery, useAction } from '@wasp/rpc' // Wasp uses a thin wrapper around react-query
+import { getTasks } from '@wasp/rpc/queries'
+import { createTask, updateTask, deleteTasks } from '@wasp/rpc/actions'
+import waspLogo from './waspLogo.png'
+import type { Task, User } from '@wasp/entities'
```

Comment:
Let's discuss the new imports here.

Here's the relevant discussion on discord: https://discord.com/channels/686873244791210014/686873245428875297/1150783903909556244.

Reply:
I believe you and I @sodic ended up kind of settling on https://discord.com/channels/686873244791210014/1150783903909556244/1151106131301769238, right?

Reply:
That's possible.

I'm fine with almost all scoping options we've discussed. The only thing I care about is reducing the number of default imports/exports.

Reply:
Done with #1696